ql/src/test/queries/clientpositive/groupingset_grouping_udf_view.q

```diff
@@ -0,0 +1,129 @@
+CREATE DATABASE IF NOT EXISTS test_grouping_set;
+CREATE TABLE test_grouping_set.test_grouping(
+id string,
+s_outlook string,
+m_outlook string,
+f_outlook string,
+s_rating string,
+m_rating string,
+f_rating string)
+CLUSTERED BY (
+id)
+INTO 2 BUCKETS
```

Comment:
Does cluster by contributes to the bug? If not could you please remove it.

Reply:
Current bug not related to clustered by, actually the reproduction queries I took from JIRA issue. Anyway I have a simple repro queries at the end so I will keep only that query and remove this query.